rtil/src/threads/ue/rebo/livesplit.rs

```diff
@@ -188,6 +241,31 @@ impl Layout {
         LIVESPLIT_STATE.lock().unwrap().layout.general_settings().clone()
     }
 }
+impl LiveSplitTime {
+    pub fn get_digits_format_and_accuracy() -> (LiveSplitDigitsFormat, LiveSplitAccuracy) {
+        let livesplit_state = LIVESPLIT_STATE.lock().unwrap();
+        (livesplit_state.digits_format, livesplit_state.accuracy)
+    }
+    pub fn set_digits_format_and_accuracy(digits_format: LiveSplitDigitsFormat, accuracy: LiveSplitAccuracy) {
```

Comment:
Does it make sense to get and set both at the same time, or should they each be 2 separate functions?